components/raftstore-v2/src/operation/ready/mod.rs

```diff
@@ -71,13 +71,33 @@ use crate::{
     worker::tablet,
 };
 
-const PAUSE_FOR_REPLAY_GAP: u64 = 128;
+const PAUSE_FOR_REPLAY_GAP: u64 = 0;
+
+pub struct InvokeClosureOnDrop(pub Option<Box<dyn FnOnce() + Send + Sync>>);
```

Comment:
This can be put into tikv_util component

Reply:
Done